security/tests/integration/nodeAgentTestEnv/integration_test.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2017 Istio Authors
```

Comment:
if you name the folder nodeAgentTest**Env**, then it makes more sense to put _env.go under this folder.

Reply:
Renamed directories and files